Header

pxntelle opened the pull request.

pxntelle committed:
header current done

pxntelle committed:
mob and tablet header-menu done

yanaskey reviewed (commented):
Відписала в особисті в тг

pxntelle committed:
html added

pxntelle committed:
html done

pxntelle committed:
Merge branch 'main-hero' into header

pxntelle committed:
new css class added

pxntelle committed:
Merge branch 'main' into header

pxntelle committed:
Merge branch 'main' into header

pxntelle committed:
icon size updated, border-bottom fixed, fonts and sizes updated

pxntelle committed:
Merge branch 'header'

yanaskey reviewed (commented):
Потрібно змінити розмір іконок соц мереж та close-btn  - їх розміри маюь співпадати з розміром посилання в яке вони загорнуті

при 768рх - не відображаються іконки соц мереж, відстань між шапкою та елементами меню-ліст, а також між самими елементами має бути 240рх та 40рх відповідно

елементи меню-ліст при усіх розширеннях мають летер-спейсінг 0.05ем

yanaskey committed:
Update header-rest.html

pxntelle committed:
header-rest html finished, social links active, icon size updated, logo fixed center positioned

pxntelle committed:
icon sizes, logo center position

pxntelle committed:
Merge branch 'main' of https://github.com/yanaskey/project_26

pxntelle committed:
menu-responsive-container created (for border-bottom) -> header container put inside (for keeping elements fixed)

зараз я спробую змерджити стилі з хедер-рест (бо там вони повніші) в хедер і гляну, чи зможу вручну вирішити конфлікти

pxntelle committed:
Merge branch 'header-rest' into header

pxntelle committed:
conflicts solved, menu-close btn position updated

header.css is complete

pxntelle committed:
soc icons = links size

pxntelle committed:
header-main fixed

pxntelle committed:
header-main fixed, social links added
